debian/copyright

```diff
@@ -0,0 +1,9 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: ev3dev-lang-python
```

Comment:
Since the debian packaging is included in upstream, we don't need the `Upstream-*` and `Source` fields here.